### DISCUSSION TITLE
Option to permanently show map north-up when routing to a location

### DISCUSSION THREAD

**Discussion by markcellus** [organicmaps/organicmaps#1348] [Ideas]
When using the Android mobile app, I prefer for the map to always point north--even when routing. But every time I start a new route to a location, I have to keep changing the map to point north always. Can there be an option to always point map north when routing so that I don't have to switch each time? Thanks!

**Comment by biodranik** [comment]
Do you want to set a default mode "north up" always when starting navigation mode?

Is it easier to use for you? Do other nav apps have a similar option?

One of possible solutions would be to save a previous manually set mode.

**Reply by markcellus** [reply]
Yes. It's in Google Maps as "Keep map north up" option in Navigation Settings. And "Lock North-up Mode" setting in Waze.

Saving a previous manually set mode is a good temporary solution, but I'd rather have an explicit option. I have my phone on my dash while driving, so I'd prefer if someone couldn't just turn it off accidentally.

**Comment by outbacker** [comment]
Osmand also supports this functionality and it's one of the main reasons I haven't switched over to organic maps yet. :)

**Comment by biodranik** [comment]
Please create an issue for that, and let's save a previous mode instead of introducing a new setting.

**Reply by markcellus** [reply]
Create an issue for what? This _is_ the issue. If you'd rather the title of this issue be changed to better reflect the way the issue will be resolved, I'll leave that up to the repo owners. 

I'm not too familiar with the codebase so I'm not sure of the details of how this should be handled. But I'd just like there be option to toggle it like other maps are doing. See my earlier comment in this thread for my reasoning.

**Reply by outbacker** [reply]
I think @biodranik means an issue that's listed as a bug/feature request instead of an idea/discussion.
Personally I'd be happy with just saving the previous mode to not clutter the settings UI. Also Google Maps is doing it the same way, which would make it easier to use for new users accustomed to that workflow.
@markcellus Would you like to open the new issue or should I do so?

**Reply by markcellus** [reply]
I see. I still dont see why the maintainers cant just open the Github issue themselves if they prefer this be one. But feel free.

**Reply by biodranik** [reply]
Because we're very, very busy :)

**Reply by markcellus** [reply]
Then do it when you can. No one's rushing you or asking for this by mid day tomorrow. 🙂

**Reply by generalmanager** [reply]
I've created #1691 hope that helps.
